src/metabase/pulse/render.clj

```diff
@@ -50,7 +52,8 @@
 (s/defn ^:private make-description-if-needed :- (s/maybe common/RenderedPulseCard)
   [dashcard]
   (when *include-description*
-    (when-let [description (-> dashcard :visualization_settings :card.description)]
+    (when-let [description (or (get-in dashcard [:visualization_settings :card.description])
+                               (db/select-one-field :description Card :id (:card_id dashcard)))]
```

Comment:
`render-pulse-card`, which calls this function, already has `card` as an argument. Could you pass that into here rather than reading it from the DB again?